Lab03-WordGuessGame/Program.cs

```diff
@@ -7,7 +7,7 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine();
             OpenWordBank();
         }
 
```

Comment:
Because you have used the full, absolute path to your text file on line 16, I cannot run your program on my machine; my computer tries to find that location to set it equal to `path` and can't, so it crashes.